helmfile/charts/networkpolicy-common/templates/rook-ceph/osd-prepare.yaml

```diff
@@ -0,0 +1,42 @@
+{{ if .Values.rookCeph.enabled }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-rook-ceph-osd-prepare
+  namespace: rook-ceph
+spec:
+  policyTypes:
+    - Ingress
```

Comment:
```suggestion
```

Reply:
@robinAwallace Thanks , this has been fixed now :)